CHANGES.rst

```diff
@@ -325,6 +325,8 @@ astropy.table
   object was not releasing the memory due to a reference cycle
   in the column ``info`` attributes. [#6277, #7448]
 
+- Fixed a bug in add_columns (issue #7539) where ``rename_duplicate=True`` would cause an error if there were in fact no duplicates. [#7540]
```

Comment:
Could you please move the entry down to the 2.0.8 section?

Reply:
Also break the line at a reasonable point, matching the formatting of other changelog entries.

Reply:
And say
```
.. bug in ``add_columns`` method where ..
```
Finally, drop the `in fact`.  Best to keep the language simple and direct here.

Reply:
Thanks for your comments. I have made the requested changes. 
May I ask, why did I need to put the changelog entry in the 2.0.8 section? I see zero other entries down there, but a whole bunch up in the 3.1 section (which was why I assumed that was where it belonged).